eng/Versions.props

```diff
@@ -30,7 +30,7 @@
     <MicrosoftDeveloperControlPlanewindowsarm64Version>0.12.9</MicrosoftDeveloperControlPlanewindowsarm64Version>
     <!-- Other -->
     <MicrosoftDotNetRemoteExecutorVersion>9.0.0-beta.25164.2</MicrosoftDotNetRemoteExecutorVersion>
-    <MicrosoftDotNetXUnitExtensionsVersion>9.0.0-beta.25164.2</MicrosoftDotNetXUnitExtensionsVersion>
+    <MicrosoftDotNetXUnitV3ExtensionsVersion>10.0.0-beta.25178.1</MicrosoftDotNetXUnitV3ExtensionsVersion>
```

Comment:
does this cause us to require .NET 10 to run tests? I don't think we want that.

Reply:
It doesn't require .NET 10. It's only because I added the package on Arcade very recently (https://github.com/dotnet/arcade/pull/15668) and that's the version of packages that Arcade currently publishes.

Reply:
OK